neuralprophet/time_net.py

```diff
@@ -392,11 +398,11 @@ def trend(self, t, quantile=None):
             trend = self.trend_k0 * torch.unsqueeze(t, dim=1)
         else:
             trend = self._piecewise_linear_trend(t)
-        if quantile is None:
-            return self.bias + trend
-        else:
+        if quantile is not None:
```

Comment:
can we skip this and do quantile-indexing in the function calling this (outside)?